apps/src/storage/dataBrowser/LibraryTable.jsx

```diff
@@ -25,6 +27,22 @@ class LibraryTable extends React.Component {
       collapsed: !this.state.collapsed
     });
 
+  importTable = () => {
+    if (this.props.current) {
+      // TODO: Implement current tables (see STAR-615)
+    } else {
+      FirebaseStorage.copyStaticTable(
+        this.props.name,
+        () => {
+          console.log('success');
```

Comment:
Is there a plan to update these success/error responses?

Reply:
Yes, we will probably want to make some UI corresponding to success/error. I don't think that flow is finalized yet, though.

Reply:
Filed https://codedotorg.atlassian.net/browse/STAR-671 and https://codedotorg.atlassian.net/browse/STAR-672 to track